lib/ui/view/organization/organization_view/organization_viewmodel.dart

```diff
@@ -132,15 +134,22 @@ class OrganizationViewModel extends BaseViewModel {
 
     try {
       setBusy(true);
-      var orgId = currentOrgId ?? '';
+      var orgId = currentOrgId ?? '61459d8e62688da5302acdb1';
+      // print('${currentOrgId} currentOrgId');
+
+      // print('Organization id from user service ${_userService.currentOrgId}');
 
       if (orgId.isNotEmpty) {
         final orgMemberList = await api.getOrganizationMemberList(orgId);
 
-        if (orgMemberList.isNotEmpty) {
-          storageService.setString(
-              StorageKeys.organizationMemberList, jsonEncode(orgMemberList));
+        // print('Organization member list length ${orgMemberList.data.length}');
```

Comment:
Comments should be an explanation of your codes.
Commenting codes should be temporal - unless of course you will use it later in the app.

Reply:
okay, thank you. changes made.